uobtheatre/productions/fixtures.json

```diff
@@ -1,146 +1,148 @@
 [
-    {
```

Comment:
There seem to be a lot of linting changes here. Have you run precommit? I think theres a linter for json in there somewhere... We should make CI fail in future